apps/ensadmin/src/components/recent-registrations/components.tsx

```diff
@@ -48,20 +75,36 @@ interface RecentRegistrationsProps {
 export function RecentRegistrations({
   ensIndexerConfig,
   indexingStatus,
+  error,
+  maxRows = DEFAULT_MAX_ROWS,
```

Comment:
```suggestion
  maxRecords = DEFAULT_MAX_RECORDS
```

Goal: I see now we already had terminology for `maxRecords`. We should align use of that terminology through all related code.